src/test/erc4626/ERC4626.t.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.10;
+
+import {Vm} from '../../../lib/forge-std/src/Vm.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {DefiBridgeProxy} from './../../aztec/DefiBridgeProxy.sol';
+import {RollupProcessor} from './../../aztec/RollupProcessor.sol';
+import {IERC4626} from './../../bridges/erc4626/Interfaces/IERC4626.sol';
+import {VaultBridge} from './../../bridges/erc4626/VaultBridge.sol';
+import {AztecTypes} from './../../aztec/AztecTypes.sol';
+import {console} from '../console.sol';
+import '../../../lib/ds-test/src/test.sol';
+
+//Tested at block 14886873 may not work at other blocks
+contract ERC4626 is DSTest {
+    Vm vm = Vm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+
+    DefiBridgeProxy defiBridgeProxy;
+    RollupProcessor rollupProcessor;
+
+    VaultBridge vaultbridge;
+
+    IERC20 constant maple = IERC20(0x33349B282065b0284d756F0577FB39c158F935e6);
+
+    IERC4626 constant vault = IERC4626(0x4937A209D4cDbD3ecD48857277cfd4dA4D82914c);
+
+    function _aztecPreSetup() internal {
+        defiBridgeProxy = new DefiBridgeProxy();
+        rollupProcessor = new RollupProcessor(address(defiBridgeProxy));
+    }
+
+    function setUp() public {
+        _aztecPreSetup();
+
+        vaultbridge = new VaultBridge(address(rollupProcessor));
+        //vaultbridge.validateVault(address(vault));
+        rollupProcessor.setBridgeGasLimit(address(vaultbridge), 1000000);
+    }
+
+    function testVaultBridge() public {
```

Comment:
Since the bridge is supposed to work for generic erc4626 vaults it would be useful to test it on more vaults on mainnet. This shouldn't take too much work as you would simply run this test over an array of vault addresses and the corresponding tokens. Finding more vaults should be fairly straightforward using etherscan's [smart contract search](https://etherscan.io/searchcontractlist?a=all&q=erc4626).

Reply:
How many do you think would be adequate?

Reply:
I think 5 should be fine.

Reply:
Please use vaults that have different input assets (ideally with different number of decimals), e.g., USDC, Dai, Eth, WBTC.